packages/commonwealth/server/routing/router.ts

```diff
@@ -1317,6 +1324,12 @@ function setupRouter(
     '/getChainNode',
     getChainNode.bind(this, models)
   );
+  registerRoute(
+    router,
+    'get',
+    '/relatedCommunities',
+    getChainNode.bind(this, models)
+  );
```

Comment:
hmm... I think this piece is not necessary? Or at least it is messed up because you have handler defined above already for `/relatedCommunities` route

Reply:
Thanks for pointing this out